docs/patterns/observability/logs.md

```diff
@@ -0,0 +1,59 @@
+# Logs
+Open Service Mesh (OSM) collects logs that are sent to stdout by default. When enabled, Fluent Bit can collect these logs, process them and send them to an output of the user's choice such as Elasticsearch, Azure Log Analytics, BigQuery, etc.
+
+
+## Fluent Bit
+[Fluent Bit](https://fluentbit.io/) is an open source log processor and forwarder which allows you to collect data/logs and send them to multiple destinations. It can be used with OSM to forward OSM controller logs to a variety of outputs/log consumers by using its output plugins.
+
+OSM provides log forwarding by optionally deploying a Fluent Bit sidecar to the OSM controller using the `--enable-fluentbit` flag during installation. The user can then define where OSM logs should be forwarded using any of the available [Fluent Bit output plugins](https://docs.fluentbit.io/manual/v/1.4/pipeline/outputs).
+
+### Configuring Log Forwarding with Fluent Bit
+By default, the Fluent Bit sidecar is configured to simply send logs to the Fluent Bit container's stdout. If you have installed OSM with Fluent Bit enabled, you may access these logs using `kubectl logs -n osm-system <osm-controller-name> -c fluentbit-logger`. However, we recommend configuring log forwarding to your preferred output for more informative results.
+
+You may configure log forwarding to an output by following these steps _before_ you install OSM.
+
+1. Define the output plugin you would like to forward your logs to in the existing `fluentbit-configmap.yaml` file by replacing the `[OUTPUT]` section with your chosen output as described by Fluent Bit documentation [here](https://docs.fluentbit.io/manual/v/1.4/pipeline/outputs).
+
+2. The existing configuration assumes OSM is running on a kind cluster so it uses a parser that matches the CRI log format. If you are using a kubernetes distribution that causes your logs to be formatted differently, you may need to update the `[PARSER]` section and the `parser` name in the `[INPUT]` section to one of the parsers defined [here](https://github.com/fluent/fluent-bit/blob/master/conf/parsers.conf).
```

Comment:
Could the Kind part be obviated here, and just say it uses CRI log format parsing by default?